frontend/src/components/Account/editProfile.module.css

```diff
@@ -0,0 +1,101 @@
+.profileForm{
+    width: 410px;
+    height: 781.36px;
+    display: flex;
+    background-color: #FFFFFF;
+    border-radius: 6px; 
+    flex-direction: column; 
+    border: 1px solid #D7D7D7;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+    box-sizing: border-box; 
+    align-items: center;
+    /* gap: 0.75em; */
+} 
+
+.profileFieldTitles {
+    font-size: 13px;
+} 
+
+.profileFields {
+    border-radius: 6px;
+    border: 1px solid #D7D7D7;
+    width: 363.08px; 
+    height: 46.25px;
+    text-align: center;
+    font-size: 10px;
+    display: flex; 
+    align-items: center;
+}
+
+.editButton, .cancelButton, .saveButton {
+    width: 142.79px; 
+    height: 38.95px; 
```

Comment:
fractional pixels are interesting they are allowed, but just want to highlight that it's a little inconsistent. TLDR it depends on zoom level, and browsers. Obviously 1 px shift any direction isn't really that important in many cases. 
I assume you got this number from figma though so shame on ray! 

https://medium.com/kajabi-ux/css-and-sub-pixel-rendering-the-case-of-the-clipped-border-4652c5a1b5ab
https://stackoverflow.com/questions/34676263/sub-pixels-calculated-and-rendered-differently-among-browsers

No required changes, but just want you to understand the sub-pixel problem